crabs/tracker/utils/tracking.py

```diff
@@ -152,3 +153,12 @@ def prep_sort(prediction: dict, score_threshold: float) -> np.ndarray:
             pred_sort.append(bbox)
 
     return np.asarray(pred_sort)
+
+
+def save_tracking_mota_metrics(
+    tracking_output_dir: Path,
+    track_results: dict[str, Any],
+) -> None:
+    track_df = pd.DataFrame(track_results)
+    output_filename = f"{tracking_output_dir}/tracking_metrics_output.csv"
```

Comment:
Just to follow our usual convention, could we timestamp the output directory? We would need to ensure it goes in the same directory as the video output if requested (or other outputs that may be requested)

Reply:
Fixed in #229